ctapipe/io/hdf5tableio.py

```diff
@@ -161,7 +161,7 @@ class Schema(tables.IsDescription):
                     continue
 
                 if col_name in Schema.columns:
-                    self.log.warning(f"Found duplicated column {col_name}, skipping")
+                    self.log.warning("Found duplicated column %s, skipping", col_name)
```

Comment:
I like the f-strings better here, it is what we use everywhere else

Reply:
I think the only reason to ever use the old %s syntax is for lines of code that are called very often, in which case python does some optimization for logs and warnings where it  doesn't evaluate the string if the log level is lower or warning is disabled (so saves a few cpu cycles).   Here, I agree f-strings are just more readable.

Reply:
Yeah, I thought there was some benefit for logging and linters usually warn about that.
I love f-strings though, so I'm happy to revert that.